.github/workflows/main.yml

```diff
@@ -73,9 +81,9 @@ jobs:
         submodules: true
     - run: rustup component add rustfmt clippy
     - run: cargo fmt --all -- --check
-    - run: cargo clippy -p tikv-jemalloc-sys -- -D clippy::all
-    - run: cargo clippy -p tikv-jemallocator -- -D clippy::all
-    - run: cargo clippy -p tikv-jemallocator-global -- -D clippy::all
-    - run: cargo clippy -p tikv-jemalloc-ctl -- -D clippy::all
+    - run: cargo clippy -p jemalloc-sys -- -D clippy::all
```

Comment:
Does it work on the `tikv-main` branch?

Reply:
No, but `tikv-main` branch will use the old workflows.